### PR TITLE
1930-active-theme-disabled-upon-editing-and-saving

### PR DESCRIPTION
**Pull request by robinhoodie0823**
https://github.com/tokens-studio/figma-plugin/assets/103296157/8f011893-9678-4a00-9c31-bd1325a4f037

### CODE REVIEW

File: src/app/store/models/reducers/tokenState/saveTheme.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export function saveTheme(state: TokenState, data: Payload): TokenState {
   });
 
   const newActiveTheme = state.activeTheme;
-  if (isActiveTheme) {
+  if (!isActiveTheme) {
```
Comment (six7) [inline src/app/store/models/reducers/tokenState/saveTheme.ts:36]: were we just doing it wrong before? or why is the fix to this so simple? any side effects that could be caused?

**Comment by keeganedwin** [comment]
Tested and confirming that this works when:
1. Editing theme name 
2. Editing set status: source, enabled, disabled.

File: src/app/store/models/reducers/tokenState/saveTheme.ts (hunk 1 above)
Comment (robinhoodie0823) [reply src/app/store/models/reducers/tokenState/saveTheme.ts:36]: Yes. The root cause was when try to edit active theme, then it all clears its activity inside the condition.

**Review by six7 — CHANGES_REQUESTED** [review]
can you base the change on `main` and not `epic/performance` ? may be best to redo the branch given the change is so small

**Pull request closed without merging**